src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -271,6 +273,7 @@ public virtual void Lock(long position, long length)
             LockInternal(position, length);
         }
 
+        [UnsupportedOSPlatform("macos")]
```

Comment:
Presumably the ref needs to be updated, too?

Reply:
Also, what about overrides? e.g. IsolatedStorageFileStream.Lock/Unlock?

Reply:
@stephentoub excellent points (as usual). I've added missing attributes (ref & overrides) PTAL

Reply:
I am sorry, I've missed the VB files... let me fix the build and get back to you